mentors/python/a.json

```diff
@@ -62,5 +62,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I use python a lot at work, and I really enjoy helping people to learn new skills!"
-  }
+  },
+  {
+    "github_username": "Alexhans",
```

Comment:
Please fix the indentation. It should be 2 spaces more than the opening bracket `{`